package.json

```diff
@@ -63,8 +63,9 @@
     ]
   },
   "dependencies": {
-    "@carbon/charts": "0.16.17",
-    "@carbon/charts-react": "0.16.17",
+    "@carbon/charts": "https://github.com/scottdickerson/carbon-charts#carbon-charts-v0.16.26-gitpkg",
```

Comment:
What's going on here? Contribution/PR that hasn't been merged yet in the main repo?

Reply:
yeah exactly.  Still waiting on PRs to be merged